lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -542,8 +543,10 @@ def setUp(self):
         super(ThreadViewSetPartialUpdateTest, self).setUp()
         self.url = reverse("thread-detail", kwargs={"thread_id": "test_thread"})
 
-    def test_basic(self):
-        self.register_get_user_response(self.user)
+    def register_thread(self, closed=False):
```

Comment:
Rather than explicitly adding a `closed` as a parameter, use `override` (in the link below) to make it more flexible.
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/discussion_api/tests/test_api.py#L1891